hbase-common/src/main/java/org/apache/hadoop/hbase/io/compress/Compression.java

```diff
@@ -82,7 +82,11 @@ public final class Compression {
 
   // LZMA
 
+  /** Deprecated. See HBASE-28506. **/
+  @Deprecated
```

Comment:
Better add a @deprecated tag in javadoc to mention the removal cycle?

Reply:
I addressed this with c21dad8a16 and will merge assuming this is good.